Remotion/Web/Development.WebTesting/HostingStrategies/AspNetCoreHostingStrategy.cs

```diff
@@ -28,13 +27,18 @@ public class AspNetCoreHostingStrategy : IHostingStrategy
   /// </summary>
   /// <param name="testSiteLayoutConfiguration">The configuration of the layout of the used test site.</param>
   /// <param name="port">Port to be used.</param>
-  public AspNetCoreHostingStrategy (ITestSiteLayoutConfiguration testSiteLayoutConfiguration, int port)
+  /// <param name="hostname">Hostname to be used. Default is 'localhost'.</param>
+  /// <param name="useHttps">Determines whether the site is hosted using https.</param>
+  public AspNetCoreHostingStrategy (ITestSiteLayoutConfiguration testSiteLayoutConfiguration, int port, string? hostname, bool useHttps)
```

Comment:
I would make `hostname` not-null and add the `localhost` default in line 55. Having a nullable parameter here feels strange.

Reply:
line 55 always either receives a hostname or throws an error, there is no nullability there. I think we should just remove the localhost default.